ComplementaryScripts/modelCuration/scaleBioMass.m

```diff
@@ -8,19 +8,50 @@
 % 3. Rescale carbohydrate fraction (total) to have biomass add up to 1
 % 4. GAM is fitted to simulate chemostat data of S. cerevisiae at low
 %    growth rates (Van Hoek et al. 1988)
-% 
+%
+% the function also accept input values for those fraction
+% format for content should be cell arrays eg: {'protein','carbohydrate'}
+% format for fraction should be number arrary eg: [0.46 0.3]
+% the third input:model is optional
+% the fourth input:save is for the option whether to save the model at
+% the end or not, default is true
+% the fifth input:fit is for fitGAM using a aerobic glucose-limited
+% chemostat to fit GAM, default is true
 % Function adapted from SLIMEr: https://github.com/SysBioChalmers/SLIMEr
 %
-% Benjamin Sanchez. Last update: 2018-09-07
+% Benjamin Sanchez.  2018-09-07
+% Feiran Li. Last update: 2019-10-01
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
-function scaleBioMass
+function model = scaleBioMass(content,fraction,model,save,fit)
```

Comment:
@feiranl any reason here for using `scaleBioMass` for adapting the model instead of a custom function? As `scaleBioMass` was originally not intended as a function to be called from other places but a one-time fix to the model, there's several sections of the function that I don't think are needed for `anaerobicModel`, and might even introduced undesired changes to the model (in particular, lines 68-141). Let me know your thoughts and if we don't need those changes I can refactor this to a separate function.

Reply:
It is due to that we don't want to have multiple `rescalePseudoReaction`, and when you want to call the function `sumBioMass`, you need to load all the data files here. Of course we can have a separate function, if you think that is a good idea. I am okay with both ways.

Reply:
then we can isolate `rescalePseudoReaction` in a single file ;)

I think it's better for the user if we don't introduce unnecessary lines every time they need only to change e.g. the protein composition

Reply:
Okay.